sabnzbd/deobfuscate_filenames.py

```diff
@@ -240,40 +275,40 @@ def deobfuscate(nzo, filelist: List[str], usefulname: str):
         biggest_file = filelist[0]
     else:
         biggest_file = None
-
+    if not (biggest_file and os.path.isfile(biggest_file)):
```

Comment:
Cleaner to use `if not biggest_file or not os.path.exists` instead of the single `not`.